progenitor-impl/src/method.rs

```diff
@@ -633,6 +648,12 @@ impl Generator {
                         }
                     }
                     (OperationParameterType::RawBody, true) => unreachable!(),
+                    (OperationParameterType::FormPart, false) => {
```

Comment:
move these above the `(OperationParameterType::RawBody,..` case which is entirely `unreachable!()` , as it is nice to have invalid cases at the bottom